guides/plugins/apps/payment.md

```diff
@@ -135,7 +135,7 @@ Keep in mind that just by providing a `message`, the payment will default to sta
 
 Asynchronous payments are more complicated and rely on interaction with the user and therefore a redirect to the payment provider. For example, this might be an integration with _PayPal_ or _Stripe_. The following process applies:
 
-Shopware sends the first `pay` POST request which is supposed to start the payment with the payment provider. All necessary data is provided: the `order`, `orderTransaction` and a `returnUrl`, where the user should be redirected to once the payment process with the payment provider has been finished. If everything is correct and the payment process is ready to start, the response to this request should be a `redirectUrl`, where the user is redirected to by Shopware. In case the payment can't be started \(for example because of missing credentials for the shop\), this can also return a `fail` status and / or a `message`.
+Shopware sends the first `pay` POST request, which is supposed to start the payment with the payment provider. All necessary data is provided: the `order`, `orderTransaction`, and a `returnUrl`, where the user should be redirected once the payment process with the payment provider has been finished. If everything is correct and the payment process is ready to start, the response to this request must be a `redirectUrl`, where the user is redirected to by Shopware. In case the payment can't be started \(for example, because of missing credentials for the shop\), the response can return a `fail` status and/or a `message`. However, the provided `message` will be logged only, not being shown on redirect back to the shop.
```

Comment:
"However, the provided `message` will be logged only, not being shown on redirect back to the shop." What does this mean?
Can I convey it like this : "However, the provided `message` is a logged status and does not redirect back to the shop."

Reply:
The message has nothing to do with the redirect. The message is an error message, that can be logged to the shopware logs, but automatically means, if it is transmitted, that the transaction has failed. The message is not shown to the user but only put into the logs.

Maybe something like this: If you provide a message, the payment process will fail automatically and a generic error message is shown to the user. The provided message will be shown in the log files.

Reply:
Yes this sounds better.